### ISSUE TITLE
Snap not starting anymore on ubuntu 19.04

### ISSUE THREAD

**Issue by ellepdesk** [IsmaelMartinez/teams-for-linux#151]
**Describe the bug**
application no longer starts when clicking icon or running from command line

**To Reproduce**
1. Go to launcher
2. Click on 'teams for linux'
3. nothing, no error, no logs, no main window

**Expected behavior**
Team for linux opens

**Screenshots**
[empty desktop]

**Desktop (please complete the following information):**
 - OS: ubuntu 19.04 (X11)
 - Installation package: snap, 
 - Version 0.3.0 2019-05-27

**Additional context**
Worked last week
removed and re-installed snap without effect
Checking out source and running 'yarn run start' does start the application
Other snap-apps do work

**Comment by IsmaelMartinez** [comment]
Hi @ellepdesk ,

There has been no changes in application for over 22 days so I can't really understand why your issue will be happening. Other than snap been temperamental. 

I can't use snap and rely on @julian-alarcon or @benyanke to sort this sort of issues. Hopefully they can have a look at it soon. In the meantime you should be able to use the deb package. (check the release folder)

I wonder if this is related to #148

**Comment by ellepdesk** [comment]
Removed the snap package, installed the deb, works again.
Thank you for the help, I hope it's just a thing for my machine and not a bug for more people.

**Comment by jokerjam** [comment]
> Removed the snap package, installed the deb, works again.
> Thank you for the help, I hope it's just a thing for my machine and not a bug for more people.

I'm on Pop!_OS 19.04 and experiencing the same problem. I followed these steps which has resolved my issue.

**Comment by IsmaelMartinez** [comment]
Closing as snap can be sometimes a pain. There is are workarounds

**Comment by nmoller** [comment]
Each time I start the service, to get it working I must go to debug > Application > clear site data.

**To Reproduce**
Steps to reproduce the behavior:
1. Install the app through snap. The first start goes fine. Add to favourites. Hit quit.
2. Click icon to start
3. You are going to have the window with an "Initialising" header (blank window) for minutes
![image](https://user-images.githubusercontent.com/4314211/63263235-08ac5b00-c256-11e9-8b51-fee9b6a8a7bb.png)

4. See error
![image](https://user-images.githubusercontent.com/4314211/63263284-28dc1a00-c256-11e9-9675-1b95d54a2bbb.png)
5. Quit
6. Restart, go in debug mode, application > clear site data. Hit refresh

**Desktop (please complete the following information):**
 - Ubuntu 18.04 Linux 4.15.0-58-generic #64-Ubuntu
 - snap
 - Version 0.7.0

**Additional context**
Going in debug mode and clearing site data, hit refresh and it works as expected.

**Comment by IsmaelMartinez** [comment]
Did you remove your config folder as indicated in the release notes? 

Clean your application cache before starting:
https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/KNOWN_ISSUES.md#blank-page

If you still get a blank screen, clean your config location:
https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/KNOWN_ISSUES.md#config-folder-locations

Otherwise, better open a new issue  instead of putting a comment in an old one. Thanks!

**Comment by nmoller** [comment]
@IsmaelMartinez : Thanks, I did not remove config folder. After doing that, it works as expected.
Sorry... for the noise, I did not read the release notes.
